src/panoramic/cli/cli.py

```diff
@@ -183,5 +183,19 @@ def validate():
 def detect_joins(target_dataset: str, yes: bool, diff: bool, overwrite: bool):
     from panoramic.cli.command import detect_joins as detect_joins_command
 
-    if not detect_joins_command(target_dataset=target_dataset, diff=diff, overwrite=overwrite, yes=yes):
-        sys.exit(1)
+    detect_joins_command(target_dataset=target_dataset, diff=diff, overwrite=overwrite, yes=yes)
+
+
+@cli.group(name='field')
```

Comment:
[offtopic] Just a thought. We will be adding a couple of groups in the coming releases - might be a good idea to slowly start organising them into separate modules.